cmd/root.go

```diff
@@ -172,6 +172,17 @@ Instance Level Configuration
     	    my-project:us-central1:my-db-server \
     	    'my-project:us-central1:my-other-server?address=0.0.0.0&port=7000'
 
+Instance Level Configuration Parameters:
+
+    There is a parameter that may only be set at the instance, and does not have
+    a corresponding global parameter.
+
+    - unix-socket-path - UnixSocketPath is the path where a Unix socket will be
```

Comment:
Let's put this paragraph under "Instance Level Configuration" and not introduce another section.